brave/brave-core
# Removes month and year from activity_info table

Resolves https://github.com/brave/brave-browser/issues/3407
Resolves https://github.com/brave/brave-browser/issues/2911
Resolves https://github.com/brave/brave-browser/issues/2943

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- visit site and make sure that you see site in the panel
- make sure that it's shown in ac table

## Reviewer Checklist:
